pkgs/applications/version-management/git-and-tools/git-branchless/default.nix

```diff
@@ -2,11 +2,16 @@
 
 , coreutils
 , git
+, libiconv
 , ncurses
 , rustPlatform
 , sqlite
+, stdenv
+, Security
+, SystemConfiguration
 }:
 
+with lib;
```

Comment:
A top-level `with` like this can make scoping confusing.  It can be worth it sometimes, but in this case you're only using it once, so please just do `lib.optionals` and drop the `with`.

Reply:
Valid point! I've removed it and rebased the PR on current master.